### PR TITLE
Feature/unit tesing

### PR DESCRIPTION
**Pull request by evieperren**

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+  navContainers: Array.prototype.slice.call(document.querySelectorAll('.navbar__container'),0),
+  mobileMenu: document.querySelector('.navbar__container--mobile'),
+  mobileLevel1: Array.prototype.slice.call(document.querySelectorAll('.level-1'),0),
+  burgerButton: document.querySelector('.navbar__header__button--burger'),
+  searchField: document.querySelector('.navbar__search__mobile'),
+  searchButton: document.querySelector('.navbar__header__button--search')
+}
+
+
+// const dropdownContainers = Array.prototype.slice.call(document.querySelectorAll('.navbar__container--desktop'),0)
```
Comment (judit-sh) [inline src/js/main.js]: shouldn't you remove the commented out code?

File: test/burger.test.js [hunk 2]
```diff
@@ -1,22 +1,32 @@
 const burger = require('../src/js/burger')
 
 describe('Burger test suite', () => {
-  let mockBurgerButton
-  let mockMobileNav
-  let mockDropdownContainers
-  let mockNavContainers
-  let mockMobileLevel1
+  console.log(document.documentElement.querySelector('.navbar__header__button--burger'))
```
Comment (judit-sh) [inline test/burger.test.js]: shouldn't this be removed? we don't push console logging, do we?

File: src/js/main.js (hunk 1 above)
Comment (evieperren) [reply src/js/main.js]: Yes, now removed

File: test/burger.test.js (hunk 2 above)
Comment (evieperren) [reply test/burger.test.js]: yes, now removed

**Pull request merged**